### PR TITLE
Handling the upgrade procedure in 7.50+ docs

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Include an explanation how to upgrade from 7.41 to 7.50+ (that is via 77.49).

## What issues does this pull request fix or reference?
Part of RHDEVDOCS-4135

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ asciidoc:
     prod-host: che-host
     prod-id-short: che
     prod-id: eclipse-che
-    prod-last-version-pre-dwo: v7.41
+    prod-last-version-pre-dwo: 7.41
```
Comment (rolfedh) [inline antora.yml:83]: Would it make sense to review and validate the other instances of these updated attributes in your doc set to make sure the new values are correct?

File: modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Upgrading from {prod-prev-short} {prod-last-version-pre-dwo} to {prod-short} {prod-ver} on OpenShift requires first upgrading to {prod-short} 7.49.
```
Comment (rolfedh) [inline modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc]:
```suggestion
Upgrading from {prod-prev-short} {prod-last-version-pre-dwo} to {prod-short} {prod-ver} on Red Hat OpenShift requires first upgrading to {prod-short} 7.49.
```
Our docs must use the Red Hat OpenShift product family name, or the specific product name, such as Red Hat OpenShift Container Platform.

Comment (rolfedh) [reply modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc]: Where are you adding this snip? I don't see an include statement for it.

File: modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc [hunk 3]
```diff
@@ -1,130 +1,10 @@
...
-$ ./rollback.sh
-----
-
-
-.Verification
-
-* Open {prod-prev-short} dashboard and log in.
-
-====
+include::example$snip_{project-context}-upgrading-che-7-41-on-openshift.adoc[]
```
Comment (rolfedh) [inline modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc:10]: I see now. Did you validate that this include statement works? I'm unfamiliar with this use of `{project-context}-` in include statements.

File: modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+.Procedure
+
+. Upgrade {prod-prev-short} {prod-last-version-pre-dwo} to {prod-short} 7.49. See link:https://www.eclipse.org/che/docs/stable/administration-guide/upgrading-che-7-41-on-openshift/[Upgrading {prod-prev-short} {prod-last-version-pre-dwo} on OpenShift].
```
Comment (rolfedh) [inline modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc]: Same concern with OpenShift here.

File: antora.yml (hunk 1 above)
Comment (max-cx) [reply antora.yml:83]: Thank you for this suggestion, Rolfe. I already checked it.

File: modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc (hunk 3 above)
Comment (max-cx) [reply modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc:10]: Yes. I checked it when generating a preview. Thank you.

File: modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc (hunk 2 above)
Comment (max-cx) [reply modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc]: :) Thank you, Rolfe! Fixing it.

File: modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc (hunk 4 above)
Comment (max-cx) [reply modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc]: Thank you. Fixed.

**Review by rolfedh — APPROVED** [review]
/lgtm

**Comment by max-cx** [comment]
Closing this PR because this has turned out a dead-end approach due to the lack of numerical versioning of the upstream docs.

**Comment by max-cx** [comment]
New PR: https://github.com/eclipse-che/che-docs/pull/2393

**Pull request closed without merging**